weave/trace_server/object_class_util.py

```diff
@@ -128,8 +129,9 @@ def dump_object(val: BaseModel) -> dict:
     # Order matters here due to the way we calculate the digest!
     # This matches the client
     dump["_type"] = cls_name
-    for k in val.model_fields:
-        dump[k] = _general_dump(getattr(val, k))
+    d = pydantic_asdict_one_level(val)
```

Comment:
It would be good to have an inline comment here explaining as neither pydantic_asdict_one_level nor _general_dump has a docstring.

Seeing the comment above, are there any consequences of this change in terms of digest calculation?

Reply:
in fact - this is important (: After adding the new field to the test with the alias, the digests had mismatch! so this brings the server in line with the client (: